authentik/policies/password/models.py

```diff
@@ -161,3 +162,83 @@ def passes_zxcvbn(self, password: str, request: PolicyRequest) -> PolicyResult:
     class Meta(Policy.PolicyMeta):
         verbose_name = _("Password Policy")
         verbose_name_plural = _("Password Policies")
+
+
+class UniquePasswordPolicy(Policy):
```

Comment:
So usually we'd have one app per policy type. This (and API things, etc.) should move to an authentik.policies.unique_password app

Reply:
Not a problem, I'll get that moved.

I think a README at the root of `policies` explaining built-in policy is its own app would help future developers too :+1: I'll throw one in.

- [ ] Move UniquePasswordPolicy to its own app
- [ ] add README.md in the root of `policies` explaining each policy should be an app

Reply:
I'm not sure how much you've poked around the code, but we do this for pretty much everything. Each provider/source type is its own app for instance. It's not really specific to policies